### DISCUSSION TITLE
how to config to show text for splash screen

### DISCUSSION THREAD

**Discussion by rqx110** [pyinstaller/pyinstaller#8677] [PyInstaller]
from the doc:
https://pyinstaller.org/en/stable/usage.html#splash-screen-experimental
>If the splash screen is configured to show text, it will automatically (as onefile archive) display the name of the file that is currently being unpacked, this acts as a progress bar.

the document did not say how to `config to show text`

**Comment by bwoodsend** [comment]
It's just [down a bit](https://pyinstaller.org/en/stable/usage.html#the-pyi-splash-module).
